docs/dev/python/table-api-users-guide/udfs/vectorized_python_udfs.md

```diff
@@ -30,7 +30,7 @@ These Python libraries are highly optimized and provide high-performance data st
 [non-vectorized user-defined functions]({% link dev/python/table-api-users-guide/udfs/python_udfs.md %}) on how to define vectorized user-defined functions.
 Users only need to add an extra parameter `udf_type="pandas"` in the decorator `udf` to mark it as a vectorized user-defined function.
 
-**NOTE:** Python UDF execution requires Python version (3.5, 3.6 or 3.7) with PyFlink installed. It's required on both the client side and the cluster side. 
+**NOTE:** Python UDF execution requires Python version (3.5, 3.6, 3.7 or 3.8) with PyFlink installed. It's required on both the client side and the cluster side. 
```

Comment:
The corresponding content of `vectorized_python_udfs.zh.md` also needs to be modified together